test/CMakeLists.txt

```
@@ -33,6 +33,7 @@ function(tbb_add_test)
         PRIVATE
         ${TBB_CXX_STD_FLAG}
         ${TBB_WARNING_LEVEL}
+        ${TBB_ALIGNMENT_WARNING_SUPPRESS}
```

Just as idea how to get rid of this variable:
```suggestion
        # Warning suppression C4324 for ARM64: structure was padded due to alignment specifier
        $<$<STREQUAL:${MSVC_CXX_ARCHITECTURE_ID},ARM64>:/wd4325>
```

Is not it too complicated? :)

---

It really looks better, thanks for the suggestion!